numpy/distutils/conv_template.py

```diff
@@ -186,7 +186,8 @@ def parse_loop_header(loophead) :
         if nsub is None :
             nsub = size
         elif nsub != size :
-            msg = "Mismatch in number of values:\n%s = %s" % (name, vals)
+            msg = "Mismatch in number of values, %d != %d\n" \
```

Comment:
Would prefer to avoid the line continuation. How about separating into a `fmt` and do the `%` substitution in the `ValueError`?